libc/calls/metalfile.internal.h

```diff
@@ -9,6 +9,14 @@ struct MetalFile {
   size_t pos;
 };
 
+extern void *__ape_com_base;
+extern size_t __ape_com_size;
+
 COSMOPOLITAN_C_END_
 #endif /* !(__ASSEMBLER__ + __LINKER__ + 0) */
+
+#define APE_COM_NAME     "/ape.com"
+#define APE_COM_ALT_NAME "/proc/self/exe"
```

Comment:
I'd say make the alt name the only name.

Reply:
@jart : Fixed.  Thanks!